invenio_i18n/cli.py

```diff
@@ -138,7 +154,6 @@ def map_to_i18next_style(pofile):
 
 
 @group(chain=True)
-@with_appcontext
```

Comment:
why did you remove the `@with_appcontext` decorator?

Reply:
because im not using any Flask features, but i can leave it if usually follows inivenio pattern?